### DISCUSSION TITLE
Getting ConanException while building Milvus

### DISCUSSION THREAD

**Discussion by chetan-shah01** [milvus-io/milvus#36972] [Q&A and General discussion]
Hello, 

I am building the milvus and getting below issue in Conan so can you please suggest on this ?

perl util/mk-fipsmodule-cnf.pl -module providers/fips.so -section_name fips_sect -key f4556650ac31d35461610bac4ed81b1a181b2d8a43ea2854cbae22ca74560813 > providers/fipsmodule.cnf
     Can't locate Digest/SHA.pm in @INC (you may need to install the Digest::SHA module) (@INC contains: /usr/local/lib64/perl5/5.32 /usr/local/share/perl5/5.32 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5) at util/mk-fipsmodule-cnf.pl line 30.
     BEGIN failed--compilation aborted at util/mk-fipsmodule-cnf.pl line 30.
     make[1]: *** [Makefile:3341: providers/fipsmodule.cnf] Error 2
     make[1]: *** Waiting for unfinished jobs....
     ranlib libcrypto.a || echo Never mind.
     make[1]: Leaving directory '/root/.conan/data/openssl/3.1.2/_/_/build/3639dd31743af9334517609d5baf870330303e60/src'
     **make: *** [Makefile:2347: build_sw] Error 2
     openssl/3.1.2: 
     openssl/3.1.2: ERROR: Package '3639dd31743af9334517609d5baf870330303e60' build failed
     openssl/3.1.2: WARN: Build folder /root/.conan/data/openssl/3.1.2/_/_/build/3639dd31743af9334517609d5baf870330303e60/build-release
     ERROR: openssl/3.1.2: Error in build() method, line 529
     	self._make()
     while calling '_make', line 521
     	self._run_make()
     while calling '_run_make', line 498
     	self.run(" ".join(command), env="conanbuild")
     	ConanException: Error 2 while executing make -j64
     conan install failed**

**Comment by yhmo** [comment]
We didn't see this error before. Try searching the keywords on google. Conan is a package manager, milvus project uses it to manage third-party libraries.
